studio/stores/jsonSchema/auth_gotrue_config.json

```diff
@@ -113,7 +113,8 @@
     },
     "EXTERNAL_AZURE_SECRET": {
       "title": "Azure secret",
-      "type": "string"
+      "type": "string",
+      "help": "Enter the data from Value, not the Secret ID."
```

Comment:
Content wise it looks good to me -- not sure if the help json has an effect for the secrets in Auth settings -- will let someone from frontend confirm  before merging.

Reply:
I add the description on `AuthProvidersFormValidtaion.tsx` maybe you can check my new commit.